fanuc_cr7ial_moveit_config/config/kinematics.yaml

```diff
@@ -0,0 +1,5 @@
+manipulator:
+  kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin
+  kinematics_solver_search_resolution: 0.005
+  kinematics_solver_timeout: 0.005
+  kinematics_solver_attempts: 3
```

Comment:
You are probably using the `Speed` `solve_type`. You'll want to configure `Distance`. See the `trac_ik` docs.